tests/unittest/llmapi/apps/_test_openai_completions.py

```
@@ -204,6 +204,34 @@ async def test_batch_completions_streaming(async_client: openai.AsyncOpenAI,
+        async_client_with_beam_search: openai.AsyncOpenAI, model_name, prompts):
+    # test beam search with streaming
+    batch = await async_client_with_beam_search.completions.create(
+        model=model_name,
+        prompt=prompts,
+        n=2,
+        max_tokens=5,
+        temperature=0.0,
+        stream=True,
+        extra_body=dict(use_beam_search=True),
+    )
+    texts = [""] * 4  # 2 prompts × 2 beams = 4 choices
```

Ruff (0.14.5)

Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)